Source/JNA/waffle-jna/src/main/java/waffle/jaas/WindowsLoginModule.java

```diff
@@ -103,6 +107,10 @@ public void initialize(final Subject initSubject, final CallbackHandler initCall
                         .valueOf(((String) option.getValue()).toUpperCase(Locale.ENGLISH));
             } else if ("roleFormat".equalsIgnoreCase(option.getKey())) {
                 this.roleFormat = PrincipalFormat.valueOf(((String) option.getValue()).toUpperCase(Locale.ENGLISH));
+            } else if ("mapRolesFromDomainGroups".equalsIgnoreCase(option.getKey())) {
+                // Remove quotation marks as well as whitespace for robustness
+                this.domains = ((String) option.getValue()).replaceAll("(\")", "").replaceAll("(^\\s*)|(\\s*$)", "")
```

Comment:
needs tests

Reply:
There is a test for this.  I modified initialize_withOptions in  waffle-jna\src\test\java\waffle\jaas\WindowsLoginModuleTest to simultaneously test this.

Reply:
I meant specifically for the stripping part. NBD